### DISCUSSION TITLE
Optimal Data Center Cooling (Project 196)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#27] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project #196](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Optimal%20Data%20Center%20Cooling)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by mormegil108** [comment]


**Reply by robertogl** [reply]
Hi @saidor54,

Thanks for for your interest in our projects. Feel free to ask any questions here.
Good luck!

**Comment by ZhorShortcut** [comment]
Hi, I am new,
I am a postgraduate student, I am interested in this project too,
Let's begin

**Reply by robertogl** [reply]
Welcome onboard @ZhorShortcut! let us know if you need anything.

**Comment by mrlaloxi** [comment]
Hi, I am a student at Universidad Nacional de Ingeniería in Peru and I'm interested in working and collaborating on this project

**Comment by Ssamy30** [comment]
Hello!

I'm an undergraduate student and definitely interested in starting with this project!

**Reply by Murad-96** [reply]
Hi,

I'm a postgrad student, would you be interested in collaborating on this project?

**Comment by MokraneMahdi** [comment]
Hello everyone,
I am interested. I'm working on "A data-based MPC approach for enhancing energy efficiency in an air-cooled datacenter".
I'd be happy to jump on board ! Please let me know.

**Reply by robertogl** [reply]
Hi @MokraneMahdi,

Your current work sounds very much in line with the scope of this project. If you are asking other people for collaboration, I suggest you to reply directly to one of the previous conversations so that the owners will get a notification. 

Cheers,

**Reply by MokraneMahdi** [reply]
Hello Robertogl,
Happy to read you..
I suggest we have a quick meeting (zoom, teams, etc) with all the elements who are interested or working on this project, where everyone is going to introduce quickly him or herself, then we jump in all togeather..
Please let me know.
Regards,
Mokrane

**Reply by robertogl** [reply]
I do not mind the idea. Actually I thought about it several times in the past. Should we ask who will be interested in participating?

**If you are interested in participating in a virtual meeting about this project, please respond to this thread!**

**Reply by MokraneMahdi** [reply]
I am interested. It's the project I'm working on currently.

**Comment by Ssamy30** [comment]
Good evening I hope all is well,

I haven’t had the chance to reply on matlab as I’ve been extremely busy.
I’m more than excited to work with all of you, please let me know how we
can collaborate, as this is a first time I work on a collaborative project
on matlab and simulink.

Thank you,
Sammy

**Comment by Ssamy30** [comment]
Same here, I’m definitely interested in participating with everyone.

**Reply by robertogl** [reply]
Sounds great. Please reply to the original thread to help keep track of the responses. Thanks!
